tests/api/cluster/test_bucket.py

```diff
@@ -0,0 +1,23 @@
+from model_mommy import mommy
+import pytest
+
+from controlpanel.api import cluster
+
+
+@pytest.fixture(autouse=True)
+def enable_db_for_all_tests(db):
+    pass
```

Comment:
uh?

Reply:
This fixture is run for every test in this file (`autouse=True`). It requires the `pytest-django` fixture `db`, which enables database access in tests - which is disabled by default. The fixture function itself does not have to do anything, as the `db` fixture is evaluated on calling this test function.